wdl_runner/wdl_runner.py

```diff
@@ -33,7 +33,8 @@
 import json
 import logging
 import os
-import urllib2
+from urllib.request import urlopen, Request
```

Comment:
nit: google style actually doesn't allow importing functions or classes (see [here](https://google.github.io/styleguide/pyguide.html#22-imports)). OK for internal use cases, but likely an issue if we want to eventually merge this upstream.